lib/components/artist/shows/index.js

```diff
@@ -6,7 +6,7 @@ import React from 'react-native';
 const { View, Text, StyleSheet } = React;
 
 import SerifText from '../../text/serif'
-import CurrentShows from './current_shows'
+import LargeShowsList from './large_list'
```

Comment:
I wanted to show you that you can call the imported variable/namespace however you prefer, it doesn’t have to be the same as how it’s called inside the module.